src/IdTokenResponse.php

```diff
@@ -27,22 +26,34 @@ class IdTokenResponse extends BearerTokenResponse
      */
     protected $claimExtractor;
 
+    /**
+     * @var Configuration
+     */
+    private $config;
+
     public function __construct(
         IdentityProviderInterface $identityProvider,
-        ClaimExtractor $claimExtractor
+        ClaimExtractor $claimExtractor,
+        Configuration $config
     ) {
         $this->identityProvider = $identityProvider;
         $this->claimExtractor   = $claimExtractor;
+        $this->config           = $config;
     }
 
     protected function getBuilder(AccessTokenEntityInterface $accessToken, UserEntityInterface $userEntity)
     {
+        $dateTimeImmutableObject = new DateTimeImmutable();
+
         // Add required id_token claims
-        $builder = (new Builder())
+        $builder = $this->config
+            ->builder()
             ->permittedFor($accessToken->getClient()->getIdentifier())
             ->issuedBy('https://' . $_SERVER['HTTP_HOST'])
-            ->issuedAt(time())
-            ->expiresAt($accessToken->getExpiryDateTime()->getTimestamp())
+            ->issuedAt($dateTimeImmutableObject)
+            ->expiresAt($dateTimeImmutableObject->setTimestamp(
+                $accessToken->getExpiryDateTime()->getTimestamp(),
```

Comment:
There is a typo here at the end of this line, that comma is a problem because when you want to use `OpenIDConnectServer\IdTokenResponse` it will throw an error that there is an unexpected ')'. As a result you will end up with failed import.

Reply:
What php version are you using?

Reply:
I'm currently on 7.2

Reply:
You can reproduce this behavior on PHP <= 7.2 https://sandbox.onlinephpfunctions.com/code/a68cb03fb685ea031c88fda8186c4eb79af7a127

Reply:
PHP 7.2 is almost a year end-of-life, so I didn't test it.

This MR focuses on the upgrade from v3 to v4.

So to use `lcobucci/jwt` v4 we need php `^7.4` or `^8.0`.

See: https://github.com/lcobucci/jwt/blob/4.2.x/composer.json#L20

Reply:
Yes, PHP 7.2 is old but it doesn't mean that we should break XY codebases just because one comma. This package requires version >= 5.5 so it should support older versions. You set `lcobucci/jwt` constraint to `^3.4|^4.0` so you still want to support older version of PHP and I see this as a breaking change/bug for users with older version of PHP on their projects. And there is no need to use that comma there because setTimestamp expects just single argument.